HIS/his_checker.py

```diff
@@ -0,0 +1,177 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) Bao Project and Contributors. All rights reserved
+
+import os
+import sys
+import subprocess
+
+cyclomatic_treshold = 10
+comment_density = 0.2
+goto_treshold = 0
+call_treshold = 7
+callee_treshold = 5
+statements_treshold = 50
```

Comment:
We should make these variables upper case to indicate they are constants.

Reply:
Changed it to upper case